tests/bootstrap_done.mjs

```diff
@@ -20,20 +20,19 @@
  *
  *     See tests/playground.html for example usage.
  */
+if (!window.blocklyLoader) {
+  throw new Error('window.blocklyLoader found');
+}
 
-let Blockly;
-
-if (window.BlocklyLoader) {
-  // Uncompiled mode.  Use top-level await
+if (window.blocklyLoader.compressed) {
+  // Compiled mode.  Nothing more to do.
+} else {
+  // Uncompressed mode.  Use top-level await
   // (https://v8.dev/features/top-level-await) to block loading of
   // this module until goog.bootstrap()ping of Blockly is finished.
-  await window.BlocklyLoader;
-  Blockly = globalThis.Blockly;
-} else if (window.Blockly) {
-  // Compiled mode.  Retrieve the pre-installed Blockly global.
-  Blockly = globalThis.Blockly;
-} else {
-  throw new Error('neither window.Blockly nor window.BlocklyLoader found');
+  await window.blocklyLoader.done;
+  // Note that this module previously did an export default of the
+  // value returned by the blocklyLoader.done promise.  This was
+  // changed in PR #5995 because library blocks and generators cannot
+  // be accessed via that the core/blockly.js exports object.
```

Comment:
I prefer to keep it this way because I expect more changes here in the future, and I want to make sure that future Chris will understand what has changed so far (and be able to easily find the relevant PR.)